robusta_krr/main.py

```
@@ -263,6 +264,13 @@ def run_strategy(
                     help="Whether to include the severity in the output or not",
                     rich_help_panel="Output Settings",
                 ),
+                severity_threshold: Optional[Severity] = typer.Option(
+                    None,
+                    "--severity-threshold",
+                    case_sensitive=False,
+                    help="Only report workloads whose recommendation is at least this severe (GOOD, OK, WARNING or CRITICAL). By default all workloads are reported.",
+                    rich_help_panel="Output Settings",
+                ),
```

Ruff (0.16.2)

[warning] Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)